models/core/tasman_identity_base_table.sql

```diff
@@ -8,7 +8,7 @@ events_per_id as (
         id
         
     from 
-        {{ ref('identifies_events') }}
+        {{ var('identity_events') }}
         
     unpivot (
         id for id_type in (user_id, anonymous_id, email) -- future work to allow input of any ids into unpivot
```

Comment:
Should we make this CTE more generic now its going on the package hub and we want to use it for a wider variety of sources? Its now based on the Rudderstack identify events but the source can be any table even a non-event table (e.g. backend tables).

Reply:
I reckon this can be a subsequent PR and the next release :)